templates/tutorialv2/includes/headline/title.part.html

```diff
@@ -1,12 +1,25 @@
 {% load i18n %}
 {% load crispy_forms_tags %}
+{% load captureas %}
+
 
 <div class="title-block">
-    {% if show_thumbnail and content.image %}
-        <div class="content-thumbnail-group">
-            <img class="thumbnail" src="{{ content.image.physical.tutorial_illu.url }}" alt="" itemprop="thumbnailUrl">
+
+    {% if show_thumbnail %}
+        <div class="content-thumbnail-group {{ content.type|lower }}-illu">
```

Comment:
Si on upload une image avec un fond transparent, on voit l'image par défaut du contenu qui reste affichée derrière l'image transparente.

J'ai corrigé le problème avec ce changement :

```suggestion
        <div class="content-thumbnail-group {% if not content.image %}{{ content.type|lower }}-illu{% endif %}">
```

Je ne sais pas si le problème provient de cette PR ou pas, donc je te laisse choisir si tu corriges le problème dans cette PR ou dans une autre.

Reply:
Je vais corriger dans celle-là.

(Après, avoir des images avec un fond transparent pour cet usage est plutôt une mauvaise idée, notamment pour des histoires de thème sombre. Quand j'écris pour Zeste de Savoir, je m'arrange pour que le fond soit uni avec la page, mais pas transparent).

Reply:
C'est fait.